nginx-sidecar/helm/nginx-sidecar/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
+name: nginx-sidecar
+description: A library helm chart that can be used to inject an nginx sidecar pod to handle SSL termination
+
+type: application
+version: 0.1.2
+appVersion: "0.0.1"
```

Comment:
this should be the nginx version

Reply:
good point, that will make it a lot easier